.github/workflows/node.js.yml

```diff
@@ -57,4 +57,5 @@ jobs:
         cache-dependency-path: '**/package-lock.json'
     - run: npm ci
     - run: npm run build --if-present
+    - run: npm test
```

Comment:
Need to find a work around for this. When you run npm test, the test suit requires a mongoose connection. We connect to the data base using the UIR stored in the .env file which is not present on the repo.